docs/tutorials/how-tos/use-batch-transactions/send-a-single-account-batch-transaction.md

```diff
@@ -0,0 +1,162 @@
+---
+seo:
+    description: Send a Batch transaction from a single account.
+metadata:
+  indexPage: true
+labels:
+  - Batch
+  - Transactions
+---
+# Send a Single Account Batch Transaction
+
+A [Batch transaction][] allows you to group multiple transactions together and execute them as a single atomic operation.
+
+This tutorial shows you how to create a Batch transaction where a single account submits multiple transactions that either all succeed together or all fail together.
+
+## Goals
+
+By the end of this tutorial, you will be able to:
+
+- Create a Batch transaction with multiple inner transactions, signed and submitted by a single account.
+- Configure the Batch transaction to ensure atomicity, so that either all inner transactions succeed or they all fail.
```

Comment:
```suggestion
- Create a `Batch` transaction with multiple inner transactions, signed and submitted by a single account.
- Configure the `Batch` transaction to ensure atomicity, so that all inner transactions succeed or fail together.
```